integration_tests/suite/test_tenant_migration.py

```diff
@@ -0,0 +1,113 @@
+         'source_internal_context': 'default',
+         'source_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'destination_internal_context': 'default',
+         'destination_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'participants': [{'user_uuid': USER_1_UUID,
+                           'tenant_uuid': NOT_MIGRATED_TENANT,
+                           'line_id': '1',
+                           'role': 'source'}]},
+        {'id': 11,
+         'date': '2017-03-23 00:00:00',
+         'requested_context': 'default',
+         'requested_tenant_uuid': NOT_MIGRATED_TENANT,
+         'requested_internal_context': 'default',
+         'requested_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'source_internal_context': 'default',
+         'source_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'destination_internal_context': 'default',
+         'destination_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'participants': [{'user_uuid': USER_2_UUID,
+                           'tenant_uuid': NOT_MIGRATED_TENANT,
+                           'line_id': '1',
+                           'role': 'source'}]},
+        {'id': 12,
+         'date': '2017-03-23 00:00:00',
+         'requested_context': 'other',
+         'requested_tenant_uuid': NOT_MIGRATED_TENANT,
+         'requested_internal_context': 'other',
+         'requested_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'source_internal_context': 'other',
+         'source_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'destination_internal_context': 'other',
+         'destination_internal_tenant_uuid': NOT_MIGRATED_TENANT,
+         'participants': [{'user_uuid': OTHER_USER_UUID,
+                           'tenant_uuid': NOT_MIGRATED_TENANT,
+                           'line_id': '1',
+                           'role': 'source'}]}
+    ])
+    def test_tenant_migration(self):
+        base = 'https://localhost:{port}/1.0/'.format(
+            port=self.service_port(9298, 'call-logd'))
+        url = url_helpers.base_join(base, 'tenant-migration')
+
+        payload = {
+            'users': [
+                {'user_uuid': USER_1_UUID, 'tenant_uuid': USERS_TENANT},
+                {'user_uuid': USER_2_UUID, 'tenant_uuid': USERS_TENANT},
+                {'user_uuid': OTHER_USER_UUID, 'tenant_uuid': OTHER_TENANT}
+            ],
+            'contexts': [
+                {'context': 'default', 'tenant_uuid': USERS_TENANT},
+                {'context': 'other', 'tenant_uuid': OTHER_TENANT},
+            ],
+        }
+        headers = {'X-Auth-Token': MASTER_TOKEN, 'Content-Type': 'application/json'}
+        resp = requests.post(url, json=payload, headers=headers, verify=False)
+        resp.raise_for_status()
+
+        with self.database.queries() as queries:
+            with closing(queries.Session()) as session:
+                query = session.query(CallLogParticipant)
+                for clp in query.all():
```

Comment:
Avoid to use acronym when it is not inline code. `call_log_participant` or `participant`

Reply:
done